src/LEGO.AsyncAPI/Models/AsyncApiSerializableExtensions.cs

```diff
@@ -18,10 +19,25 @@ public static class AsyncApiSerializableExtensions
         /// <param name="element">The AsyncApi element.</param>
         /// <param name="stream">The output stream.</param>
         /// <param name="specificationVersion">The AsyncApi specification version.</param>
+        [Obsolete($"Please use overridden version that accepts a {nameof(AsyncApiWriterSettings)} instance.")]
         public static void SerializeAsJson<T>(this T element, Stream stream, AsyncApiVersion specificationVersion)
```

Comment:
Let's make these take a defaulted settings, instead of obsoleting.

This way we keep the breakage minimal. Most people likely use default writer settings anyway.